### ISSUE TITLE
Wrong overlay position for pages with vertical scroll

### ISSUE THREAD

**Issue by peleccom** [angular/material2#15385]
#### What is the expected behavior?
Overlay is fully visible. There is a padding between right side of overlay and page border

#### What is the current behavior?
Overlay position is wrong.

#### What are the steps to reproduce?
https://stackblitz.com/edit/angular-2nsh5d

Place mat-select at the right border
Set `overflow-y: scroll;` for `body` element 

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
angular: 7.2.7
angular-material: 7.3.1

#### Is there anything else we should know?
![image](https://user-images.githubusercontent.com/899242/53819958-fe362780-3f7b-11e9-91b8-27de8bda5556.png)

**Comment by Egorvoz** [comment]
True. Has the same bug in our project. And this is really annoying.

**Comment by josephperrott** [comment]
@crisbeto looks like the overlay positioning isn't working from the edge of the scroll bar, but from the edge of the window instead.

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
